group_vars/development

```diff
@@ -3,10 +3,10 @@ mysql_root_password: devpw
 web_user: vagrant
 
 wordpress_sites:
-  example.dev:
+  example.com:
```

Comment:
I reverted some naming updates after talking with @retlehs but it's still a little now with more descriptive comments.

@fullyint think of it in the context of deploying:

`./deploy.sh staging example.com`

`./deploy.sh production example.com`

You're still deploying the same project/site, just doing it to a different host.

Rather than: `./deploy.sh staging staging.example.com`